basket/settings.py

```diff
@@ -222,23 +218,6 @@ def path(*args):
     "watchman.checks.databases",
 )
 
-# Salesforce.com
```

Comment:
There are Salesforce-related secrets in referenced in https://github.com/mozmeao/basket-config/blob/master/mozmeao-fr/basket-prod/web-deploy.yaml, so they prob should be removed too - and once that's done and released, we can cull the actual secrets from https://github.com/mozmeao/basket-private-config

Reply:
Good plan, thanks for the pointer.

Reply:
We also need to be sure that we're using the proper sandbox variable, we may well be relying on `EXACTTARGET_USE_SANDBOX` value to set `USE_SANDBOX_BACKEND` for deployments.